src/GraphQL/Instrumentation/FieldMiddlewareBuilder.cs

```diff
@@ -28,7 +28,7 @@ public IFieldMiddlewareBuilder Use(Func<FieldMiddlewareDelegate, FieldMiddleware
             return this;
         }
 
-        private static readonly FieldMiddlewareDelegate _defaultDelegate = context => Task.FromResult(NameFieldResolver.Instance.Resolve(context));
```

Comment:
Now I see that `object`->`Task<object>` "boxing" is eliminated saving memory here for middlewares. We always use `ValueTask` container across the codebase and so in most cases work with stack memory. In this regard behaviour of resolvers is the same as middlewares - both work with `ValueTask` so we can just passthrough results.

Reply:
Exactly 🎉🎉🎉